recipes/passenger/templates/passenger.js

```diff
@@ -0,0 +1,10 @@
+// Startup script for Phusion Passenger that uses next.js cli
+// Run `blitz build` before starting
+const path = require('path')
+
+const blitzPath = path.join(__dirname, 'node_modules', 'next', 'dist', 'bin', 'next');
```

Comment:
I think we should use a `blitz` bin instead

Reply:
Yes, I tried first that approach, however it does not work because it spawns next.js bin and Passenger stucks with that and created multiple instances without mapping for itself :/

Reply:
Ok we can use the next bin for now. Only limitation is currently next bin does not support custom server but blitz build does. However, soon that will be moved into the next bin and it will fully work

Reply:
Thanks @flybayer :) Another approach I can imagine for the future that the next.js bin is not spawned but required and enhanced with the arguments blitz needs.